kds/kds.go

```diff
@@ -59,12 +61,24 @@ var (
 	OidSpl7 = asn1.ObjectIdentifier([]int{1, 3, 6, 1, 4, 1, 3704, 1, 3, 7})
 	// OidUcodeSpl is the x509v3 extension for V[CL]EK microcode security patch level.
 	OidUcodeSpl = asn1.ObjectIdentifier([]int{1, 3, 6, 1, 4, 1, 3704, 1, 3, 8})
+	// OidFmcSpl is the x509v3 extension for V[CL]EK FMC security patch level.
+	OidFmcSpl = asn1.ObjectIdentifier([]int{1, 3, 6, 1, 4, 1, 3704, 1, 3, 9})
 	// OidHwid is the x509v3 extension for VCEK certificate associated hardware identifier.
 	OidHwid = asn1.ObjectIdentifier([]int{1, 3, 6, 1, 4, 1, 3704, 1, 4})
 	// OidCspID is the x509v3 extension for a VLEK certificate's Cloud Service Provider's
 	// origin TLS key's certificate's subject key's CommonName.
 	OidCspID = asn1.ObjectIdentifier([]int{1, 3, 6, 1, 4, 1, 3704, 1, 5})
 
+	// TCBStructVersion0 is the version of the TCB structure for Milan & Genoa
+	TCBStructVersion0 = uint8(0)
+	// TCBStructVersion1 is the version of the TCB structure for Turin
+	TCBStructVersion1 = uint8(1)
```

Comment:
Struct version constants shouldn't need to be exported given the version fields aren't exported.